### PR TITLE
Fix - post/private: Password protected button is not aligned (ED-2533)

### PR DESCRIPTION
**Pull request by nisan250**
Now when the form for entering private post/page is showing the button aligned correctly with the text input for password.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
+	Requires PHP: 5.6
```
Comment (StyleShit) [inline style.css:11]: @matipojo do we need that here? @nisan250 says it was automatically generated

File: assets/scss/theme/_general.scss [hunk 2]
```diff
@@ -84,3 +84,9 @@ to override any of the settings in this section, add your styling code in the cu
 .bypostauthor {
 	font-size: inherit;
 }
+
+.post-password-form {
+	input[type='submit'] {
+		vertical-align: bottom;
+	}
+}
```
Comment (matipojo) [inline assets/scss/theme/_general.scss:92]: Add an empty line

**Review by matipojo — CHANGES_REQUESTED** [review]
Fix conflicts

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from develop

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix conflicts

**Comment by nisan250** [comment]
The issue have already fixed.

**Pull request closed without merging**